docs/mongo.md

```diff
@@ -0,0 +1,47 @@
+# MongoDB
+
+[MongoDB](https://www.mongodb.com/) is a document-based, distributed database
+
+## Using the MongoDB Infra
+
+### Customizing your CR
+
+An example to enable only the MongoDB infra (this does _not_ run a workload):
+
+```yaml
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: mongo-infra
+  namespace: ripsaw
+spec:
+  infrastructure:
+    name: mongo
+    args:
+      servers: 3 # has to be > 0
+      storageclass: "default" # uses the default sc by default if option not specified
```

Comment:
by default there is not storageclass name with name `default`. 
When storageclass is created one can specify 
`annotations:
    storageclass.kubernetes.io/is-default-class: "true"`
what will make an storage class `preferred` storageclass ( or default ) in case there are other storage classes - then one with `is-default-class` will be used. 
In this case, it will search for storageclass with name `default` and it will not find it and then  

- fail back to storage class with `is-default-class` 
- fail if there is not any with `is-default-class` configured

I propose to put here and storage class name which is always present , eg. on ec2 it is `gp2` or leave it empty and state to update this file with working storage class from where storage will be taken.

Reply:
Thanks for the review. I've updated it to empty and added a bit more detailed description.